docs/DeveloperGuide.adoc

```diff
@@ -211,10 +211,35 @@ image::activityDiagram_for_expensesplitter.png[]
 
 ==== Add Command
 
+*Implementation*
+
+The following is an activity diagram and a detailed explanation of the operations `AddItemCommand` performs. +
+
+image::AddFoodActivityDiagram.png[]
+
+1. The `AddCommand#execute(Model model)` method is executed and a new `Food` would be created.
+
+2. The `Model#addFood(Food food)` method would then call `FoodBook#addFood(Food food)`.
+
+3. The `FoodBook#addFood(Food food)` method would then add the food into a `UniqueFoodList`.
+
+4. If successful, a success message will be generated by `CommandResult` and it will be returned with the generated
+success message. Otherwise, an error message showing the correct command syntax is thrown as `CommandException`.
+
+5. If the command syntax was valid and Food was added to the FoodBook, `LogicManager` calls
+`FoodBookStorage#saveFoodBook(ReadOnlyFoodBook foodBook)` which saves the new Food
+Amount into JSON format after serializing it using `JsonAdaptedFood`.
+
+The following is a sample sequence diagram of the AddCommand.
+
+image::AddFoodSequenceDiagram.png[]
+
```

Comment:
Maybe you can add the label for the figure like `Interactions Inside the Logic Component for the addfood Command`.

Reply:
@sheexiong Alright noted thanks will add labels